cypress/e2e/signIn.cy.js

```diff
@@ -2,10 +2,29 @@
 
 describe('Sign In page', () => {
   beforeEach(() => {
+    cy.visit('/login');
+  });
+
+  const username = 'tomsmith';
+  const password = 'SuperSecretPassword!';
+
+  it('should login user with valid credentials', () => {
+    cy.loginUser(username, password);
+
+    cy.contains('h2', 'Secure Area');
+
+    cy.checkAuthByHref('Logout');
```

Comment:
```suggestion
    cy.assertButtonIsExist('Logout');
```
I would name it like this to make it more understandable

Reply:
Agree, fixed